### PR TITLE
Revert #13659 (Add a transaction context function to get the raw transaction hash)

### PR DESCRIPTION
**Pull request by junkil-park**
## Description
Revert #13659 (Add a transaction context function to get the raw transaction hash)

It turns out that we need more discussion on the requirement of `raw_transaction_hash`.

### CODE REVIEW

**Comment by georgemitenkov** [comment]
Why revert?

File: aptos-move/aptos-release-builder/src/components/feature_flags.rs [hunk 1]
```diff
@@ -131,7 +131,6 @@ pub enum FeatureFlag {
     FederatedKeyless,
     TransactionSimulationEnhancement,
     CollectionOwner,
-    TransactionContextHashFunctionUpdate,
```
Comment (igor-aptos) [inline aptos-move/aptos-release-builder/src/components/feature_flags.rs:134]: @georgemitenkov we are removing a feature flag, are you anywhere depending on index of EnableLoaderV2 directly? (as this change will change it's index)

**Comment by igor-aptos** [comment]
why revert - we are discussing whether function `raw_transaction_hash` needs to have additional arguments

**Review by igor-aptos — APPROVED** [review]

**Review by vusirikala — APPROVED** [review]

**Review by lightmark — APPROVED** [review]

File: aptos-move/aptos-release-builder/src/components/feature_flags.rs (hunk 1 above)
Comment (georgemitenkov) [reply aptos-move/aptos-release-builder/src/components/feature_flags.rs:134]: No, should be ok

**Pull request merged**